polaris-cli/package.json

```diff
@@ -31,7 +31,7 @@
   },
   "engine-strict": true,
   "engines": {
-    "node": ">=14.13.1"
+    "node": "^16.16.0 || ^18.13.0"
```

Comment:
By setting the engines on the specific node packages, they shall refuse to install if the node version does not match these versions. This is great for blocking installation on older unsupported versions, however the configuration of "^16.16.0 || ^18.13.0" means "Only node 16 or 18" - you're currently also blocking the installation of these packages on  the currently supported node 19 (and the lts node 20 when it comes out in April).

For these per-package engine fields I strongly think we should allow current and future versions for the sake of forwards compatibility - `^16.16.0 || >=18.13.0`.